src/cascade/stats/estimation.py

```diff
@@ -31,6 +31,47 @@ def meas_bounds_to_stdev(df):
     return df.drop(["meas_lower", "meas_upper"], axis=1)
 
 
+def es_to_stdev(mean_value, es):
+    """
+    Takes an array of values (like mean), and
+    effective sample size (es) and transforms it
+    to standard error assuming that the sample
+    size is Poisson distributed.
+
+    Args:
+        mean_value:
+        es:
+
+    Returns:
+
+    """
+    mean = mean_value.copy()
+    sample_size = es.copy()
+    count = mean * sample_size
+
+    # TODO: May want to include Wilson's interval
+    #  calculation instead. Only difficulty is that
+    #  we need to know the "param_type" associated
+    #  with the quantity, i.e. proportion or rate.
+
+    # Standard deviation for binomial with measure zero is approximately:
+    std_0 = 1.0 / sample_size
+
+    # When counts are >= 5, use standard deviation assuming that the
+    # count is Poisson.
+    # Note that when count = 5, mean is 5 / sample size.
+    under_5 = count < 5
+    std_5 = np.sqrt(5.0 / sample_size**2)
+
+    std = np.sqrt(mean / sample_size)
+
+    # For counts < 5, linearly interpolate between std_0 and std_5,
+    # replacing the regular standard deviation.
+    std[under_5] = ((5.0 - count[under_5]) * std_0[under_5] +
+                    count[under_5] * std_5[under_5]) / 5.0
+    return std
+
```

Comment:
I think the calculations are correct and yours is more clear than mine.  Perhaps we should just implement the Wilson code, using a keyword argument,  proportional = False, then we will not have to revisit this in the future, just change the calls to the function.

Also we should find out if we have the proportional info available from anywhere.

Reply:
- [ ] Wilson's Score Interval